src/discussions/common/AuthorLabel.test.jsx

```diff
@@ -90,6 +90,15 @@ describe('Author label', () => {
       },
     );
 
+    it(
+      'it is not clickable when enableInContextSidebar is true',
+      async () => {
+        renderComponent(author, authorLabel, linkToProfile, labelColor, true);
+
+        expect(screen.queryByTestId('learner-posts-link')).not.toBeInTheDocument();
+      },
+    );
+
```

Comment:
Combine these two test cases using test.each and provide more appropriate descriptions for better clarity and readability

Reply:
Both these test cases are testing two different scenarios. can not be coming with test.each